modules/bridge/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Fixed
```

Comment:
As this is a hotfix we need to also include the changelog version/date here.
```suggestion
## 4.0.2 - 2022-03-22

### Fixed
```

Reply:
wouldn't that happen when the hotfix is released? 🤔

Reply:
No... nothing automatically adds changelog dates at this stage, it's all manual.

Reply:
should the hotfix version be `4.0.1` not `4.0.2`?

Reply:
No, look at the package.json and what's already been released. As the `package.json` is already 4.0.1 then it means that version has already been released, so this will be `4.0.2` when it's released.